### PR TITLE
Update to 1.0/5.0 launch

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,48 +11,46 @@ More details about aspect cli plugins is on the [plugin documentation].
...
 ## Releasing
 
 Just push a tag to your GitHub repo.
 The actions integration will create a release.
 
 [bazelisk]: https://bazel.build/install/bazelisk
-[aspect cli]: https://aspect.build
-[plugin documentation]: https://aspect-build.github.io/aspect-cli/help/topics/plugins
+[aspect cli]: https://aspect.build/cli
+[plugin documentation]: https://docs.aspect.build/aspect-build/aspect-cli/5.0.1/docs/help/topics/plugins.html
```
Comment (gregmagolan) [inline README.md:55]: guess we still need to add special handling for `latest` version

File: release/release.bzl [hunk 2]
```diff
@@ -2,6 +2,27 @@
 """
 
 load("@io_bazel_rules_go//go:def.bzl", "go_binary")
+load("@bazel_skylib//lib:dicts.bzl", "dicts")
+load("@bazel_tools//tools/build_defs/hash:hash.bzl", "tools", _sha256 = "sha256")
+
+def _sha256_impl(ctx):
+    out = _sha256(ctx, ctx.file.artifact)
```
Comment (gregmagolan) [inline release/release.bzl:9]: ohh. that is simpler than what I had in silo

File: README.md (hunk 1 above)
Comment (alexeagle) [reply README.md:55]: Yeah I filed the issue on the docs repo

**Review by gregmagolan — APPROVED** [review]
🦖

**Pull request merged**